crates/biome_css_formatter/tests/specs/css/casing.css.snap

```diff
@@ -72,8 +71,8 @@ div.classNames#AND_Ids.ArePreserved {
 }
 
 div {
-	--preserved-casing: blue;
-	color: var(--Preserved-Casing);
+	--Preserved-Casing: blue;
+	color: VAR(--Preserved-Casing);
```

Comment:
It might be a good rule to highlight incorrect cases for widely used functions.
cc @togami2864